internal/service/fw/model_atcfw_access_code_rule.go

```diff
@@ -0,0 +1,107 @@
+package fw
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/fw"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type AtcfwAccessCodeRuleModel struct {
+	Action       types.String `tfsdk:"action"`
+	Data         types.String `tfsdk:"data"`
+	Description  types.String `tfsdk:"description"`
+	RedirectName types.String `tfsdk:"redirect_name"`
+	Type         types.String `tfsdk:"type"`
+}
+
+var AtcfwAccessCodeRuleAttrTypes = map[string]attr.Type{
+	"action":        types.StringType,
+	"data":          types.StringType,
+	"description":   types.StringType,
+	"redirect_name": types.StringType,
+	"type":          types.StringType,
+}
+
+var AtcfwAccessCodeRuleResourceSchemaAttributes = map[string]schema.Attribute{
```

Comment:
Can you please add description for these fields?

Reply:
Added